_includes/tutorials/count-messages/ksql/markup/dev/03d-count-all-messages-pageview.adoc

```diff
@@ -0,0 +1,15 @@
+Now let's count all of the messages in the topic: 
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/count-messages/ksql/code/tutorial-steps/dev/03d-count-all-messages-pageview.sql %}</code></pre>
++++++
+
+We're specifying a `LIMIT 1` clause here so that the query exits after the first row has returned. Without this, ksqlDB will continue to write any changes in the count to the screen as new messages arrive. 
+
+You should now see a count of all the messages in the topic: 
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/count-messages/ksql/code/tutorial-steps/dev/03d-count-all-messages-pageview.out %}</code></pre>
++++++
+
+NOTE: _You may be wondering about the purpose of `X` in the query. It's a dummy field to pursuade ksqlDB to do an aggregation across all messages. You can read more about this https://github.com/confluentinc/ksql/issues/430[here]_.
```

Comment:
For your consideration - it's not obvious from the rendered formatting that users can click on `here`, so maybe change to make it more obvious  but I don't have a better suggestion

Reply:
fixed. thanks.